the_wizz/pair_maker.py

```diff
@@ -99,22 +103,23 @@ def run(self, reference_catalog, unknown_catalog, use_unkn_weights=False):
         Parameters
         ----------
         reference_catalog : 'dict' of `numpy.ndarray`
-            Dictionary containing arrays 
+            Dictionary containing arrays
```

Comment:
Make sure to rebase, I put in a lot more doc strings and changed this file slightly.